File: fibonacci.hs
fibonacci :: Integer -> Integer
fibonacci n
  | n <= 1    = n
  | otherwise = fibonacci (n - 1) + fibonacci (n - 2)

main :: IO ()
main = do
  putStrLn "Digite um número:"
  input <- getLine
  let n = read input :: Integer
  putStrLn ("O " ++ show n ++ "-ésimo número da sequência de Fibonacci é " ++ show (fibonacci n))

File: fibonacci.js
function fibonacci(n) {
  if (n <= 1) return n;
  return fibonacci(n - 1) + fibonacci(n - 2);
}

const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

readline.question("Digite um número: ", input => {
  const n = parseInt(input);
  console.log(`O ${n}-ésimo número da sequência de Fibonacci é ${fibonacci(n)}`);
  readline.close();
});
